cmd/node-cache/app/cache_app.go

```diff
@@ -126,26 +122,6 @@ func newIPTables() utiliptables.Interface {
 	return utiliptables.New(execer, dbus, utiliptables.ProtocolIpv4)
 }
 
-func (c *CacheApp) setupNetworking() error {
-	var err error
-	clog.Infof("Setting up networking for node cache")
-	err = c.netifHandle.AddDummyDevice(c.params.InterfaceName)
```

Comment:
Just to make sure, so I assume the `EnsureDummyDevice()` func in the periodic check would have the same effect as this, hence not needing it any more?

Reply:
Yes, Ensure will call AddDummyDevice if it is not already present.